tests/load/locustfiles/load.py

```diff
@@ -57,13 +64,17 @@ def __init__(self):
     def tick(self) -> TickTuple | None:
         """Override defining the desired distribution for Autopush load testing.
 
-        Returns:
-            TickTuple: Distribution parameters
-                user_count: Total user count
-                spawn_rate: Number of users to start/stop per second when changing
-                            number of users
-                user_classes: None or a List of user classes to be spawned
-            None: Instruction to stop the load test
+        Returns
+        -------
+        TickTuple | None
+
+        Distribution Parameters
```

Comment:
I've not heard of the term 'Distribution Parameters' before. For clarity `user_count` `spawn_rate` and `user_classes` are the fields in the return type `TickTuple`, having them defined at the same indentation as `None`, the alternative return type to TickTuple I think may be confusing in future.

Reply:
I will take it out, even though it was there before??? Let me know if the change to just enumerate the contained parameters is clearer.

```python
    def tick(self) -> TickTuple | None:
        """Override defining the desired distribution for Autopush load testing.

        Returns:
            TickTuple | None 

            TickTuple Contained Parameters
                user_count: Total user count
                spawn_rate: Number of users to start/stop per second when changing
                            number of users
                user_classes: None or a List of user classes to be spawned
                
            None: Instruction to stop the load test
        """
```